src/poetry/utils/env.py

```diff
@@ -1637,9 +1640,12 @@ def get_pip_command(self, embedded: bool = False) -> list[str]:
         ]
 
     def get_supported_tags(self) -> list[Tag]:
-        output = self.run_python_script(GET_SYS_TAGS)
+        if self.get_version_info() < (3, 0, 0):
+            raise PoetryException("Poetry does not support python 2.7 environments")
```

Comment:
We can use richer text here, even indicating the plugin. Either via a simple console exception or via `logger.error`.

Reply:
By richer text you mean more info (including info about plugin)?